subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/JavaInstallationMetadata.java

```diff
@@ -77,4 +77,13 @@ public interface JavaInstallationMetadata {
      */
     @Internal
     Directory getInstallationPath();
+
+    /**
+     * Returns true if this installation corresponds to the currently running JVM.
```

Comment:
Shall we call this maybe "build JVM"? I am not sure though, current JVM also sounds fine.

Reply:
Yep, "build JVM" sounds better. Fixed!